.github/azure-pipelines.yaml

```diff
@@ -0,0 +1,54 @@
+trigger:
+  branches:
+    include:
+    - master
+  tags:
+    include:
+    - ^v\d+\.\d+(\.\d+)?(-\S*)?$
```

Comment:
I think we should also double check the regex. `^v?(\d+(?:\.\d+)*)$` seems more like what we want.

Reply:
Yeah I'm not too good with those. FYI `semver` provides some recommended regex [here,](https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string) but it might be a bit complex for our needs -- includes things like `rc` and other stuff.

Reply:
The one included there is the one Neil sent me which was recommended by travis. I can change it.

Reply:
I couldn't get the current one to work. [Here's](https://regex101.com/r/57BCQu/1) how I tested the one I posted. It might not be exactly what we want though.

Reply:
Okay, never mind. I just tried the current one and it works perfectly. I must have copied it incorrectly.

Reply:
Which one do you mean as "current"...?

This: `^v?(\d+(?:\.\d+)*)$`

or: `^v\d+\.\d+(\.\d+)?(-\S*)?$`

Reply:
Thanks for changing it back. Sorry about that.